homestar-runtime/src/metrics.rs

```diff
@@ -0,0 +1,19 @@
+//! Collect metrics and monitoring endpoint for Prometheus
```

Comment:
minor. `.` after Prometheus.

Reply:
Also, endpoint is not here right (other file).

Reply:
Updated to `Collect metrics and setup recorder and scrape endpoint.` 👍